website/docs/d/is_security_groups.html.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_groups"
+description: |-
+  Get information about SecurityGroupCollection
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_groups
+
+Provides a read-only data source for SecurityGroupCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_security_groups" "example" {
+}
+
+OR with Filters:
+
+Filter with VPC name
+data "ibm_is_security_groups" "example" {
+  vpc_name = "<Name of VPC>"
+}
+
+Filter with VPC ID
+data "ibm_is_security_groups" "example" {
+  vpc_id = "<VPC identifier>"
+}
+
+Filter with VPC CRN
+data "ibm_is_security_groups" "example" {
+  vpc_crn = "<VPC CRN>"
+}
+
+Filter with Resource Group ID
+data "ibm_is_security_groups" "example" {
+  resource_group="<Resource Group ID>"
+}
+```
+
+
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+- `id` - The unique identifier of the SecurityGroupCollection.
```

Comment:
optional/required  needs to be specified here

Reply:
Need confirmation on this, please.

Reply:
done.